dev/changelog.yml

```
@@ -0,0 +1,15 @@
+types:
```

For consistency since `project` is also singular:
```suggestion
type:
```